src/components/ButtonCheckMovie.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ButtonCheckMovie from './ButtonCheckMovie';
+
+describe('ButtonCheckMovie', () => {
+  it('renders', () => {
```

Comment:
"it renders" 😄 
what does it render?  :D

Reply:
I have written it more precisely now :)